CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+## [v0.13.21](https://github.com/chef/chef-dk/tree/v0.13.21) (2016-04-15)
```

Comment:
I chose to call it just `v12.9` in the Chef CHANGELOG. I think we should do the same here but am open to discussion.